services/slam/builtin/orbslam_yaml.go

```diff
@@ -137,7 +137,11 @@ func (slamSvc *builtIn) orbGenYAML(ctx context.Context, cam camera.Camera) error
 
 	// TODO change time format to .Format(time.RFC3339Nano) https://viam.atlassian.net/browse/DATA-277
 	timeStampNow := time.Now().UTC().Format(slamTimeFormat)
-	saveMapName := filepath.Join(slamSvc.dataDirectory, "map", slamSvc.cameraName+"_data_"+timeStampNow)
+
+	// remove colons from camera name(in case of remote)
```

Comment:
It would be more general to remove all non-alphanumeric characters (https://gosamples.dev/remove-non-alphanumeric/), and then check that the resulting string is non empty. But it's fine with me if you want to merge this for the demo, if you make a ticket to change it later.